Fail to get yarn configuration. /home/cbel/Desktop/YIHAN/venv/lib/python3.8/site-packages/jupyterlab/staging/yarn.js:4
(()=>{var Qge=Object.create;var AS=Object.defineProperty;var bge=Object.getOwnPropertyDescriptor;var Sge=Object.getOwnPropertyNames;var vge=Object.getPrototypeOf,xge=Object.prototype.hasOwnProperty;var J=(r=>typeof require<"u"?require:typeof Proxy<"u"?new Proxy(r,{get:(e,t)=>(typeof require<"u"?require:e)[t]}):r)(function(r){if(typeof require<"u")return require.apply(this,arguments);throw new Error('Dynamic require of "'+r+'" is not supported')});var Pge=(r,e)=>()=>(r&&(e=r(r=0)),e);var w=(r,e)=>()=>(e||r((e={exports:{}}).exports,e),e.exports),ut=(r,e)=>{for(var t in e)AS(r,t,{get:e[t],enumerable:!0})},Dge=(r,e,t,i)=>{if(e&&typeof e=="object"||typeof e=="function")for(let n of Sge(e))!xge.call(r,n)&&n!==t&&AS(r,n,{get:()=>e[n],enumerable:!(i=bge(e,n))||i.enumerable});return r};var Pe=(r,e,t)=>(t=r!=null?Qge(vge(r)):{},Dge(e||!r||!r.__esModule?AS(t,"default",{value:r,enumerable:!0}):t,r));var QK=w((GXe,BK)=>{BK.exports=wK;

SyntaxError: Unexpected token {
    at createScript (vm.js:80:10)
    at Object.runInThisContext (vm.js:139:10)
    at Module._compile (module.js:616:28)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)
    at Function.Module._load (module.js:497:3)
    at Function.Module.runMain (module.js:693:10)
    at startup (bootstrap_node.js:188:16)
    at bootstrap_node.js:609:3

[I 2025-03-11 00:28:44.587 ServerApp] jupyter_lsp | extension was successfully linked.
[I 2025-03-11 00:28:44.590 ServerApp] jupyter_server_terminals | extension was successfully linked.
[I 2025-03-11 00:28:44.593 ServerApp] jupyterlab | extension was successfully linked.
[I 2025-03-11 00:28:44.596 ServerApp] notebook | extension was successfully linked.
[I 2025-03-11 00:28:44.597 ServerApp] Writing Jupyter server cookie secret to /home/cbel/.local/share/jupyter/runtime/jupyter_cookie_secret
[I 2025-03-11 00:28:44.710 ServerApp] notebook_shim | extension was successfully linked.
[I 2025-03-11 00:28:44.750 ServerApp] notebook_shim | extension was successfully loaded.
[I 2025-03-11 00:28:44.751 ServerApp] jupyter_lsp | extension was successfully loaded.
[I 2025-03-11 00:28:44.752 ServerApp] jupyter_server_terminals | extension was successfully loaded.
[I 2025-03-11 00:28:44.753 LabApp] JupyterLab extension loaded from /home/cbel/Desktop/YIHAN/venv/lib/python3.8/site-packages/jupyterlab
[I 2025-03-11 00:28:44.753 LabApp] JupyterLab application directory is /home/cbel/Desktop/YIHAN/venv/share/jupyter/lab
[I 2025-03-11 00:28:44.753 LabApp] Extension Manager is 'pypi'.
[I 2025-03-11 00:28:44.770 ServerApp] jupyterlab | extension was successfully loaded.
[I 2025-03-11 00:28:44.772 ServerApp] notebook | extension was successfully loaded.
[I 2025-03-11 00:28:44.773 ServerApp] Serving notebooks from local directory: /home/cbel/Desktop/YIHAN/unet
[I 2025-03-11 00:28:44.773 ServerApp] Jupyter Server 2.14.2 is running at:
[I 2025-03-11 00:28:44.773 ServerApp] http://localhost:8888/tree?token=9f276798b6fd05f2f04663f6639d67d49bdbbec144d0b914
[I 2025-03-11 00:28:44.773 ServerApp]     http://127.0.0.1:8888/tree?token=9f276798b6fd05f2f04663f6639d67d49bdbbec144d0b914
[I 2025-03-11 00:28:44.773 ServerApp] Use Control-C to stop this server and shut down all kernels (twice to skip confirmation).
[C 2025-03-11 00:28:44.774 ServerApp] 
    
    To access the server, open this file in a browser:
        file:///home/cbel/.local/share/jupyter/runtime/jpserver-6533-open.html
    Or copy and paste one of these URLs:
        http://localhost:8888/tree?token=9f276798b6fd05f2f04663f6639d67d49bdbbec144d0b914
        http://127.0.0.1:8888/tree?token=9f276798b6fd05f2f04663f6639d67d49bdbbec144d0b914
[I 2025-03-11 00:28:45.151 ServerApp] Skipped non-installed server(s): bash-language-server, dockerfile-language-server-nodejs, javascript-typescript-langserver, jedi-language-server, julia-language-server, pyright, python-language-server, python-lsp-server, r-languageserver, sql-language-server, texlab, typescript-language-server, unified-language-server, vscode-css-languageserver-bin, vscode-html-languageserver-bin, vscode-json-languageserver-bin, yaml-language-server
[W 2025-03-11 00:45:10.299 ServerApp] 404 GET /hub/api (@127.0.0.1) 10.05ms referer=None
[I 2025-03-11 00:45:10.301 ServerApp] 302 GET / (@127.0.0.1) 0.22ms
[I 2025-03-11 00:45:10.303 JupyterNotebookApp] 302 GET /tree? (@127.0.0.1) 0.33ms
[W 2025-03-11 00:45:26.645 ServerApp] 401 POST /login? (@127.0.0.1) 0.59ms referer=None
[W 2025-03-11 00:45:26.649 ServerApp] wrote error: 'Forbidden'
    Traceback (most recent call last):
      File "/home/cbel/Desktop/YIHAN/venv/lib/python3.8/site-packages/tornado/web.py", line 1788, in _execute
        result = method(*self.path_args, **self.path_kwargs)
      File "/home/cbel/Desktop/YIHAN/venv/lib/python3.8/site-packages/tornado/web.py", line 3289, in wrapper
        url = self.get_login_url()
      File "/home/cbel/Desktop/YIHAN/venv/lib/python3.8/site-packages/jupyter_server/base/handlers.py", line 783, in get_login_url
        raise web.HTTPError(403)
    tornado.web.HTTPError: HTTP 403: Forbidden
[W 2025-03-11 00:45:26.650 ServerApp] 403 GET /api/kernelspecs?1741625126647 (@127.0.0.1) 1.04ms referer=None
[W 2025-03-11 00:45:26.650 ServerApp] wrote error: 'Forbidden'
    Traceback (most recent call last):
      File "/home/cbel/Desktop/YIHAN/venv/lib/python3.8/site-packages/tornado/web.py", line 1788, in _execute
        result = method(*self.path_args, **self.path_kwargs)
      File "/home/cbel/Desktop/YIHAN/venv/lib/python3.8/site-packages/tornado/web.py", line 3289, in wrapper
        url = self.get_login_url()
      File "/home/cbel/Desktop/YIHAN/venv/lib/python3.8/site-packages/jupyter_server/base/handlers.py", line 783, in get_login_url
        raise web.HTTPError(403)
    tornado.web.HTTPError: HTTP 403: Forbidden
[W 2025-03-11 00:45:26.651 ServerApp] 403 GET /api/kernels?1741625126648 (@127.0.0.1) 0.47ms referer=None
[W 2025-03-11 00:45:26.652 ServerApp] wrote error: 'Forbidden'
    Traceback (most recent call last):
      File "/home/cbel/Desktop/YIHAN/venv/lib/python3.8/site-packages/tornado/web.py", line 1788, in _execute
        result = method(*self.path_args, **self.path_kwargs)
      File "/home/cbel/Desktop/YIHAN/venv/lib/python3.8/site-packages/tornado/web.py", line 3289, in wrapper
        url = self.get_login_url()
      File "/home/cbel/Desktop/YIHAN/venv/lib/python3.8/site-packages/jupyter_server/base/handlers.py", line 783, in get_login_url
        raise web.HTTPError(403)
    tornado.web.HTTPError: HTTP 403: Forbidden
[W 2025-03-11 00:45:26.652 ServerApp] 403 GET /api/kernels?1741625126649 (@127.0.0.1) 0.60ms referer=None
[W 2025-03-11 00:45:26.652 ServerApp] wrote error: 'Forbidden'
    Traceback (most recent call last):
      File "/home/cbel/Desktop/YIHAN/venv/lib/python3.8/site-packages/tornado/web.py", line 1788, in _execute
        result = method(*self.path_args, **self.path_kwargs)
      File "/home/cbel/Desktop/YIHAN/venv/lib/python3.8/site-packages/tornado/web.py", line 3289, in wrapper
        url = self.get_login_url()
      File "/home/cbel/Desktop/YIHAN/venv/lib/python3.8/site-packages/jupyter_server/base/handlers.py", line 783, in get_login_url
        raise web.HTTPError(403)
    tornado.web.HTTPError: HTTP 403: Forbidden
[W 2025-03-11 00:45:26.652 ServerApp] 403 GET /api/sessions?1741625126650 (@127.0.0.1) 0.88ms referer=None
[W 2025-03-11 00:45:26.657 ServerApp] 404 GET /hub/api (@127.0.0.1) 0.75ms referer=None
[W 2025-03-11 00:45:26.657 ServerApp] wrote error: 'Forbidden'
    Traceback (most recent call last):
      File "/home/cbel/Desktop/YIHAN/venv/lib/python3.8/site-packages/tornado/web.py", line 1788, in _execute
        result = method(*self.path_args, **self.path_kwargs)
      File "/home/cbel/Desktop/YIHAN/venv/lib/python3.8/site-packages/tornado/web.py", line 3289, in wrapper
        url = self.get_login_url()
      File "/home/cbel/Desktop/YIHAN/venv/lib/python3.8/site-packages/jupyter_server/base/handlers.py", line 783, in get_login_url
        raise web.HTTPError(403)
    tornado.web.HTTPError: HTTP 403: Forbidden
[W 2025-03-11 00:45:26.657 ServerApp] 403 GET /api/kernelspecs?1741625126654 (@127.0.0.1) 1.02ms referer=None
[I 2025-03-11 00:45:26.658 ServerApp] 302 GET / (@127.0.0.1) 0.19ms
[I 2025-03-11 00:45:26.660 JupyterNotebookApp] 302 GET /tree? (@127.0.0.1) 0.29ms
[C 2025-03-11 00:47:53.189 ServerApp] received signal 15, stopping
[I 2025-03-11 00:47:53.191 ServerApp] Shutting down 5 extensions
Fail to get yarn configuration. /home/cbel/Desktop/YIHAN/venv/lib/python3.8/site-packages/jupyterlab/staging/yarn.js:4
(()=>{var Qge=Object.create;var AS=Object.defineProperty;var bge=Object.getOwnPropertyDescriptor;var Sge=Object.getOwnPropertyNames;var vge=Object.getPrototypeOf,xge=Object.prototype.hasOwnProperty;var J=(r=>typeof require<"u"?require:typeof Proxy<"u"?new Proxy(r,{get:(e,t)=>(typeof require<"u"?require:e)[t]}):r)(function(r){if(typeof require<"u")return require.apply(this,arguments);throw new Error('Dynamic require of "'+r+'" is not supported')});var Pge=(r,e)=>()=>(r&&(e=r(r=0)),e);var w=(r,e)=>()=>(e||r((e={exports:{}}).exports,e),e.exports),ut=(r,e)=>{for(var t in e)AS(r,t,{get:e[t],enumerable:!0})},Dge=(r,e,t,i)=>{if(e&&typeof e=="object"||typeof e=="function")for(let n of Sge(e))!xge.call(r,n)&&n!==t&&AS(r,n,{get:()=>e[n],enumerable:!(i=bge(e,n))||i.enumerable});return r};var Pe=(r,e,t)=>(t=r!=null?Qge(vge(r)):{},Dge(e||!r||!r.__esModule?AS(t,"default",{value:r,enumerable:!0}):t,r));var QK=w((GXe,BK)=>{BK.exports=wK;

SyntaxError: Unexpected token {
    at createScript (vm.js:80:10)
    at Object.runInThisContext (vm.js:139:10)
    at Module._compile (module.js:616:28)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)
    at Function.Module._load (module.js:497:3)
    at Function.Module.runMain (module.js:693:10)
    at startup (bootstrap_node.js:188:16)
    at bootstrap_node.js:609:3

[I 2025-03-11 00:50:18.991 ServerApp] jupyter_lsp | extension was successfully linked.
[I 2025-03-11 00:50:18.994 ServerApp] jupyter_server_terminals | extension was successfully linked.
[I 2025-03-11 00:50:18.997 ServerApp] jupyterlab | extension was successfully linked.
[I 2025-03-11 00:50:19.000 ServerApp] notebook | extension was successfully linked.
[I 2025-03-11 00:50:19.116 ServerApp] notebook_shim | extension was successfully linked.
[I 2025-03-11 00:50:19.156 ServerApp] notebook_shim | extension was successfully loaded.
[I 2025-03-11 00:50:19.157 ServerApp] jupyter_lsp | extension was successfully loaded.
[I 2025-03-11 00:50:19.158 ServerApp] jupyter_server_terminals | extension was successfully loaded.
[I 2025-03-11 00:50:19.159 LabApp] JupyterLab extension loaded from /home/cbel/Desktop/YIHAN/venv/lib/python3.8/site-packages/jupyterlab
[I 2025-03-11 00:50:19.159 LabApp] JupyterLab application directory is /home/cbel/Desktop/YIHAN/venv/share/jupyter/lab
[I 2025-03-11 00:50:19.159 LabApp] Extension Manager is 'pypi'.
[I 2025-03-11 00:50:19.176 ServerApp] jupyterlab | extension was successfully loaded.
[I 2025-03-11 00:50:19.178 ServerApp] notebook | extension was successfully loaded.
[I 2025-03-11 00:50:19.179 ServerApp] Serving notebooks from local directory: /home/cbel/Desktop/YIHAN/unet
[I 2025-03-11 00:50:19.179 ServerApp] Jupyter Server 2.14.2 is running at:
[I 2025-03-11 00:50:19.179 ServerApp] http://localhost:8888/tree?token=f554aa01734dcd0b0be6163dced349074ff6153d20e82bdf
[I 2025-03-11 00:50:19.179 ServerApp]     http://127.0.0.1:8888/tree?token=f554aa01734dcd0b0be6163dced349074ff6153d20e82bdf
[I 2025-03-11 00:50:19.179 ServerApp] Use Control-C to stop this server and shut down all kernels (twice to skip confirmation).
[C 2025-03-11 00:50:19.180 ServerApp] 
    
    To access the server, open this file in a browser:
        file:///home/cbel/.local/share/jupyter/runtime/jpserver-7069-open.html
    Or copy and paste one of these URLs:
        http://localhost:8888/tree?token=f554aa01734dcd0b0be6163dced349074ff6153d20e82bdf
        http://127.0.0.1:8888/tree?token=f554aa01734dcd0b0be6163dced349074ff6153d20e82bdf
[I 2025-03-11 00:50:19.515 ServerApp] Skipped non-installed server(s): bash-language-server, dockerfile-language-server-nodejs, javascript-typescript-langserver, jedi-language-server, julia-language-server, pyright, python-language-server, python-lsp-server, r-languageserver, sql-language-server, texlab, typescript-language-server, unified-language-server, vscode-css-languageserver-bin, vscode-html-languageserver-bin, vscode-json-languageserver-bin, yaml-language-server
[W 2025-03-11 00:52:10.969 ServerApp] 404 GET /hub/api (@127.0.0.1) 12.21ms referer=None
[I 2025-03-11 00:52:10.970 ServerApp] 302 GET / (@127.0.0.1) 0.22ms
[I 2025-03-11 00:52:10.971 JupyterNotebookApp] 302 GET /tree? (@127.0.0.1) 0.33ms
[I 2025-03-11 00:52:20.728 ServerApp] User bff5b615b982451093a8f0d7a8f409cf logged in.
[I 2025-03-11 00:52:20.729 ServerApp] 302 POST /login? (bff5b615b982451093a8f0d7a8f409cf@127.0.0.1) 3.27ms
[I 2025-03-11 00:54:52.359 ServerApp] Kernel started: 1de8e015-989c-4517-9652-22e55ae10bab
[I 2025-03-11 00:54:52.719 ServerApp] Connecting to kernel 1de8e015-989c-4517-9652-22e55ae10bab.
2025-03-11 00:54:54.034457: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:939] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2025-03-11 00:54:54.038939: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:939] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2025-03-11 00:54:54.039420: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:939] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2025-03-11 00:54:57.548870: I tensorflow/core/platform/cpu_feature_guard.cc:151] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN) to use the following CPU instructions in performance-critical operations:  AVX2 FMA
To enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.
2025-03-11 00:54:57.549501: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:939] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2025-03-11 00:54:57.550338: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:939] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2025-03-11 00:54:57.551052: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:939] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2025-03-11 00:54:57.894727: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:939] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2025-03-11 00:54:57.895250: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:939] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2025-03-11 00:54:57.895710: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:939] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2025-03-11 00:54:57.896144: I tensorflow/core/common_runtime/gpu/gpu_process_state.cc:214] Using CUDA malloc Async allocator for GPU: 0
2025-03-11 00:54:57.896578: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1525] Created device /job:localhost/replica:0/task:0/device:GPU:0 with 12288 MB memory:  -> device: 0, name: NVIDIA GeForce RTX 3090, pci bus id: 0000:01:00.0, compute capability: 8.6
2025-03-11 00:55:22.839236: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:939] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2025-03-11 00:55:39.690911: I tensorflow/stream_executor/cuda/cuda_dnn.cc:366] Loaded cuDNN version 8202
[I 2025-03-11 06:49:25.150 ServerApp] Starting buffering for 1de8e015-989c-4517-9652-22e55ae10bab:bb5da35c-7060-4bac-89d4-c6812122499e
[I 2025-03-11 09:08:17.145 ServerApp] 302 GET / (@127.0.0.1) 0.32ms
[I 2025-03-11 09:08:17.154 JupyterNotebookApp] 302 GET /tree? (@127.0.0.1) 0.55ms
[I 2025-03-11 09:13:40.740 ServerApp] User 30d25a1301a34f2399e0190fbfbe1a3d logged in.
[I 2025-03-11 09:13:40.740 ServerApp] 302 POST /login? (30d25a1301a34f2399e0190fbfbe1a3d@127.0.0.1) 0.67ms
[I 2025-03-11 09:13:40.778 ServerApp] Connecting to kernel 1de8e015-989c-4517-9652-22e55ae10bab.
[I 2025-03-11 09:13:40.778 ServerApp] Discarding 12923 buffered messages for 1de8e015-989c-4517-9652-22e55ae10bab:bb5da35c-7060-4bac-89d4-c6812122499e
[I 2025-03-11 09:48:57.208 ServerApp] 302 GET / (@127.0.0.1) 0.35ms
[I 2025-03-11 09:48:57.215 JupyterNotebookApp] 302 GET /tree? (@127.0.0.1) 0.45ms
[I 2025-03-11 09:49:22.606 ServerApp] User 19a97b06aef2463fbbc9063c6c96c752 logged in.
[I 2025-03-11 09:49:22.606 ServerApp] 302 POST /login? (19a97b06aef2463fbbc9063c6c96c752@127.0.0.1) 0.70ms
[I 2025-03-11 09:49:22.638 ServerApp] Connecting to kernel 1de8e015-989c-4517-9652-22e55ae10bab.
[I 2025-03-11 13:38:52.650 ServerApp] 302 GET / (@127.0.0.1) 0.24ms
[I 2025-03-11 13:38:52.653 JupyterNotebookApp] 302 GET /tree? (@127.0.0.1) 0.31ms
[I 2025-03-11 13:39:07.885 ServerApp] User 22f3bbd745974a1983c72e3a75b3d9bb logged in.
[I 2025-03-11 13:39:07.885 ServerApp] 302 POST /login? (22f3bbd745974a1983c72e3a75b3d9bb@127.0.0.1) 0.80ms
[I 2025-03-11 13:39:07.931 ServerApp] Connecting to kernel 1de8e015-989c-4517-9652-22e55ae10bab.
[I 2025-03-11 14:49:17.758 ServerApp] 302 GET / (@127.0.0.1) 0.40ms
[I 2025-03-11 14:49:17.762 JupyterNotebookApp] 302 GET /tree? (@127.0.0.1) 0.51ms
[I 2025-03-11 14:49:20.130 ServerApp] User 33bba1d7d9dd43499beed71bda4ea8be logged in.
[I 2025-03-11 14:49:20.130 ServerApp] 302 POST /login? (33bba1d7d9dd43499beed71bda4ea8be@127.0.0.1) 0.77ms
[I 2025-03-11 14:49:20.209 ServerApp] Connecting to kernel 1de8e015-989c-4517-9652-22e55ae10bab.
[I 2025-03-11 17:00:40.274 ServerApp] Starting buffering for 1de8e015-989c-4517-9652-22e55ae10bab:9fa68e71-b93e-4dd9-b1c3-f9e662d6a86d
[I 2025-03-11 17:01:12.634 ServerApp] 302 GET / (@127.0.0.1) 0.40ms
[I 2025-03-11 17:01:12.639 JupyterNotebookApp] 302 GET /tree? (@127.0.0.1) 0.51ms
[I 2025-03-11 17:01:16.953 ServerApp] User b19d2d58b977450fbc0c738ae438cc2a logged in.
[I 2025-03-11 17:01:16.953 ServerApp] 302 POST /login? (b19d2d58b977450fbc0c738ae438cc2a@127.0.0.1) 0.66ms
[I 2025-03-11 17:01:16.997 ServerApp] Connecting to kernel 1de8e015-989c-4517-9652-22e55ae10bab.
[I 2025-03-11 17:01:49.630 ServerApp] Kernel interrupted: 1de8e015-989c-4517-9652-22e55ae10bab
[I 2025-03-11 23:44:10.217 ServerApp] Kernel interrupted: 1de8e015-989c-4517-9652-22e55ae10bab
[I 2025-03-11 23:47:12.041 ServerApp] 302 GET / (@127.0.0.1) 0.26ms
[I 2025-03-11 23:47:12.044 JupyterNotebookApp] 302 GET /tree? (@127.0.0.1) 0.33ms
[I 2025-03-11 23:48:19.450 ServerApp] User 9ec7cbe9a2e747d3a0fe0ea6c2647cd2 logged in.
[I 2025-03-11 23:48:19.451 ServerApp] 302 POST /login? (9ec7cbe9a2e747d3a0fe0ea6c2647cd2@127.0.0.1) 0.65ms
[I 2025-03-11 23:48:19.485 ServerApp] Connecting to kernel 1de8e015-989c-4517-9652-22e55ae10bab.
[I 2025-03-12 07:03:12.319 ServerApp] Starting buffering for 1de8e015-989c-4517-9652-22e55ae10bab:71b39151-e76c-4402-9aa3-ff4310df4c3b
[I 2025-03-12 08:44:08.627 ServerApp] 302 GET / (@127.0.0.1) 0.29ms
[I 2025-03-12 08:44:08.632 JupyterNotebookApp] 302 GET /tree? (@127.0.0.1) 0.35ms
[I 2025-03-12 08:44:18.956 ServerApp] User 4273db01d99447ab9ede71967d42b576 logged in.
[I 2025-03-12 08:44:18.956 ServerApp] 302 POST /login? (4273db01d99447ab9ede71967d42b576@127.0.0.1) 0.57ms
[I 2025-03-12 08:44:18.985 ServerApp] Connecting to kernel 1de8e015-989c-4517-9652-22e55ae10bab.
[I 2025-03-12 08:45:05.428 ServerApp] Kernel interrupted: 1de8e015-989c-4517-9652-22e55ae10bab
[I 2025-03-12 16:00:58.864 ServerApp] Starting buffering for 1de8e015-989c-4517-9652-22e55ae10bab:ed7382fe-08d8-4d68-b190-2d0e07deedea
[I 2025-03-12 22:04:16.312 ServerApp] 302 GET / (@127.0.0.1) 0.27ms
[I 2025-03-12 22:04:16.314 JupyterNotebookApp] 302 GET /tree? (@127.0.0.1) 0.36ms
[I 2025-03-12 22:04:17.781 ServerApp] User 082898f272544b2f8e7e033a70d895bc logged in.
[I 2025-03-12 22:04:17.781 ServerApp] 302 POST /login? (082898f272544b2f8e7e033a70d895bc@127.0.0.1) 0.56ms
[I 2025-03-12 22:04:17.812 ServerApp] Connecting to kernel 1de8e015-989c-4517-9652-22e55ae10bab.
